go.mod

```diff
@@ -1,9 +1,8 @@
 module github.com/docker/hub-tool
 
-go 1.15
+go 1.16
```

Comment:
Do we need to bump the minimum Go version?

Reply:
Probably not :sweat_smile: I'll revert.